src/theme/index.hbs

```diff
@@ -111,7 +111,7 @@
                 <hr>
                 <p><a href="https://github.com/hexatomic/hexatomic/releases/latest"><button style="background-color: #4183c4; border: none; color: white; padding: 12px 30px; cursor: pointer; width: 100%"><i class="fa fa-download"></i> Download Hexatomic</button></a></p>
                 <p><i class="fa fa-copyright"></i> 2018ff. <a href="https://github.com/orgs/hexatomic/teams/project/members" target="_blank">Hexatomic project team</a></p>
-                <p><i class="fa fa-envelope"></i> hexatomic [at] corpus-tools.org</p>
+                <p><a href="https://sympa.cms.hu-berlin.de/sympa/info/hexatomic-users"><button style="background-color: #00a847; border: none; color: white; padding: 12px 30px; cursor: pointer; width: 100%"><i class="fa fa-envelope"></i> Mailing list</button></a></p>
```

Comment:
I think it would be better to have the large flat buttons together, so first download, then mailing list and the other things like members below

Reply:
Why not.